packages/digitalocean-management/gitops.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: digitalocean-management
+spec:
+  description: Management DOKS cluster resources
+  clusterResourceWhitelist:
+    - group: '*'
+      kind: '*'
+  destinations:
+    - namespace: '*'
+      server: '*'
+  sourceRepos:
+    - https://github.com/kubezero/kubezero
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: digitalocean-management-infrastructure
+  annotations:
+    argocd.argoproj.io/sync-wave: '100'
+    argocd.argoproj.io/depends-on: digitalocean-provider
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: digitalocean-management-infrastructure
```

Comment:
There is no project with this name `digitalocean-management-infrastructure` it should be adjusted to `digitalocean-management`